### PR TITLE
Feature/pass test

### PR DESCRIPTION
**Pull request by arlen34**

### CODE REVIEW

File: src/main/java/kg/peaksoft/bilingualb6/dto/request/PassTestRequest.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.List;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
```
Comment (adilet524) [inline src/main/java/kg/peaksoft/bilingualb6/dto/request/PassTestRequest.java:13]: You can use it without NoArgsConstructor

Comment (arlen34) [reply src/main/java/kg/peaksoft/bilingualb6/dto/request/PassTestRequest.java:13]: ok

File: src/main/java/kg/peaksoft/bilingualb6/service/PassTestService.java [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        }
+        float finalScore = 0f;
+        for (QuestionAnswer questionScore : questionAnswers) {
+            finalScore += questionScore.getScore();
+        }
+        result.setFinalScore(finalScore);
+        resultRepository.save(result);
+        return new SimpleResponse("Test is complete!", "PASS");
+    }
+}
```
Comment (adilet524) [inline src/main/java/kg/peaksoft/bilingualb6/service/PassTestService.java:141]: Everything is okay, Good job!

File: src/main/java/kg/peaksoft/bilingualb6/dto/response/TestResponseForClient.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package kg.peaksoft.bilingualb6.dto.response;
+
+import lombok.*;
+
+@Getter
+@Setter
+@NoArgsConstructor
```
Comment (adilet524) [inline src/main/java/kg/peaksoft/bilingualb6/dto/response/TestResponseForClient.java:7]: Write here without NoArgsConstructor

Comment (arlen34) [reply src/main/java/kg/peaksoft/bilingualb6/dto/response/TestResponseForClient.java:7]: ok

File: src/main/java/kg/peaksoft/bilingualb6/service/PassTestService.java (hunk 2 above)
Comment (arlen34) [reply src/main/java/kg/peaksoft/bilingualb6/service/PassTestService.java:141]: > Everything is okay, Good job!

yes, everything works with a bang

**Pull request closed without merging**